### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by amentetteh**
Opened for

Basic UI with plain HTML implemented
Remove button  basic configuration
Add button basic configuration
Local storage configured

Kindly review this project for us please

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            <div class="field">
+              <input type="text"
+                class="form-control" name="booktitle" id="booktitle" aria-describedby="Title" placeholder="Title">
+            </div>
+            <div class="field">
+              <input type="text"
+                class="form-control" name="bookauthor" id="bookauthor" aria-describedby="Author" placeholder="Author">
+            </div>
+                <input type="submit" class="form-control button" name="addbutton" id="addbutton" aria-describedby="Add" value="Add">
+        </form>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to validate your form so the user can be able to add a new book with no `author` or `title`

    - ### Local storage is storing a book with no `title` or `author`
   image

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      booksTable.appendChild(
+        createBookRow(bookList[i].Name, bookList[i].Author),
+      );
+    }
+  }
+};
+
+const addBook = (name, author) => {
+  const books = JSON.parse(localStorage.getItem('books')) || [];
+  books.push({ Name: name, Author: author });
```
Comment (levy002) [inline js/index.js]: - [ ] Kindly make sure each book has a unique `id` which will help to make sure that you are removing the book you want. :wink:

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @gealsanchez  & @amentetteh   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make the user can be able to see the list of all books and remove it as well.

    _**This how your app is working**_
   chrome-capture-2022-10-1 (1)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 2 above)
Comment (amentetteh) [reply js/index.js]: Thanks very much @levy002 . Should we add this id also on the UI ?

Comment (amentetteh) [reply js/index.js]: ok

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi team, 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**